scripts/build/Dockerfile.aarch64-cross.hdr

```diff
@@ -0,0 +1,5 @@
+FROM docker.io/dockcross/base:latest
+
+ENV ARCH=aarch64
+ENV DEBIAN_ARCH=arm64
+ENV CROSS_TRIPLE=aarch64-linux-gnu
```

Comment:
Nit:  maybe we can update `TRIPLE` to commonly used `TRIPLET` as we already change things here?
Otherwise looks good to me

Reply:
done